spec/bundler/cli_spec.rb

```diff
@@ -51,6 +51,22 @@
     end
   end
 
+  context "when ENV['BUNDLE_GEMFILE'] is specified" do
+    before { create_file "Gemfile.dev" }
```

Comment:
You don't need to `create_file`, `gemfile` will create it for us.

Reply:
Done